source/components/StyledArticleTagsSelector/getAccentColor.js

```diff
@@ -0,0 +1,5 @@
+import get from 'lodash/get';
+
+export default function getAccentColor(theme) {
+    return get(theme, 'site.accent') || get(theme, 'color.fandom_aqua');
```

Comment:
Instead of doing this I'll probably add accent color prop to main component and use this instead. IDK.

Reply:
I think instead of using lodash we should start using the native approach with babel. These strings make the code less useful to IDE. What do you think?

Reply:
Yeah. Additionally, I also talked to some GitHub people we probably should provide the theme in a different way, but I'll work on that post-PTO.